lib/web/apiserver.go

```diff
@@ -1280,11 +1273,12 @@ func (h *Handler) newSessionResponse(ctx *SessionContext) (*CreateSessionRespons
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
 	return &CreateSessionResponse{
 		TokenType:              roundtrip.AuthBearer,
 		Token:                  token.GetName(),
 		TokenExpiresIn:         int(token.Expiry().Sub(ctx.parent.clock.Now()) / time.Second),
-		SessionInactiveTimeout: int(h.webIdleTimeout.Milliseconds()),
+		SessionInactiveTimeout: int(ctx.session.GetInactiveTimeout().Milliseconds()),
```

Comment:
This appears to be the only place we're not using a duration, and it isn't immediately obvious that the timeout here needs to be in milliseconds. Internally, a duration is measured in nanoseconds, so I'd be a little concerned that we might mix this up in the future.

Can we make this harder to mess up by either renaming the field to something that includes milliseconds (perhaps `SessionInactiveTimeoutMillis`) or at least updating the doc on that field to mention that it is expressed in milliseconds?

Reply:
renamed and also updated field comment